site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -161,6 +164,10 @@ method.
 
 ## Implementation details
 
+### web_accessible_resources
+
+When an extension uses declarativeNetRequest APIs to redirect a public resource request to a resource that is not web accessible, it is blocked and will result in a net::ERR_UNSAFE_REDIRECT error. The above holds true even if the resource that is not web accessible is owned by the redirecting extension. To declare resources for use with declarativeNetRequest APIs, the `"web_accessible_resources"` array must be declared and populated in the manifest as documented [here](/docs/extensions/mv3/manifest/web_accessible_resources/).
```

Comment:
>net::ERR_UNSAFE_REDIRECT

This looks like C++ syntax, which means an extension developer will never see this string. This needs to be stated in terms of what an extension developer will see.